### DISCUSSION TITLE
more_itertools should be added to requirements.txt

### DISCUSSION THREAD

**Discussion by duerra** [meisnate12/Plex-Meta-Manager#2308] [General]
While installing Kometa, I discovered that a missing additional module needed to be installed, _more_itertools_

**Comment by chazlarson** [comment]
What context and platform?

My current linux install doesn't have that module installed and it's running fine.

I ran tests on OS X with Python:
```
  3.9.19
  3.10.14
  3.11.9
  3.12.6
```

And none of them complained about `more_itertools` when running `kometa.py`.  I didn't run anything that talked to any external services, so if you have a log showing where this is happening it would be useful.

**Reply by duerra** [reply]
The installation was initiated from a TrueNAS jail.

**Reply by duerra** [reply]
Unfortunately I no longer have the logs, but the app errored out after attempting to run the python script following the initial install.  After investigation, I identified the cause as needing to additionally install more_itertools.

**Reply by chazlarson** [reply]
I cannot reproduce this, so probably some idiosyncracy related to the truenas jail.  There is no code in Kometa that attempts to import `more_itertools` directly, in any case.
